<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPhotoDlg</class>
 <widget class="QDialog" name="AddPhotoDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <widget class="QLabel" name="lbSelectedFile">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>70</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Select file</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leSelectedFile">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="tbSelectPhoto">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbNewFileName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>70</width>
        <height>13</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>New file name without extension</string>
      </property>
      <property name="text">
       <string>New file name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leNewFileName">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>160</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbImage">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>140</width>
        <height>140</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string>x</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lbTitle">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leTitle">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbDescr">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>70</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="teDescr">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>170</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbYear">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>242</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Year</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbYear">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>240</y>
        <width>52</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>2099</number>
      </property>
     </widget>
     <widget class="QLabel" name="lbMonth">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>242</y>
        <width>35</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Month</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbMonth">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>240</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>12</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbDay">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>240</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>31</number>
      </property>
     </widget>
     <widget class="QLabel" name="leDay">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>242</y>
        <width>35</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Day</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="sbLat">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>270</y>
        <width>70</width>
        <height>22</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="decimals">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLabel" name="lbLat">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>272</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Lat</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="sbLon">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>270</y>
        <width>70</width>
        <height>22</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="decimals">
       <number>5</number>
      </property>
      <property name="minimum">
       <double>-180.000000000000000</double>
      </property>
      <property name="maximum">
       <double>180.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="lbLon">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>272</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Lon</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddPhotoDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddPhotoDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
